operations/security_and_authentication/Audit_Logs.md

```diff
@@ -0,0 +1,110 @@
+# Audit Logs
+
+## Enable or disable audit logging for host and Kubernetes APIs
+
+Audit logs are used to monitor the system and search for suspicious behavior.
+Host and Kubernetes API audit logging can be enabled to produce extra audit logs for analysis.
+Enabling audit logging is optional. If enabled it generates some load and data on the non-compute nodes \(NCNs\).
+By default, host and Kubernetes API audit logging are not enabled.
+
+### Procedure
+
+The method for updating the audit log settings varies depending on the state of the system.
+
+1. Select one of the following options based on the installation status of the system:
```

Comment:
The "options" have options of their own, and can use a little clarity to help the admin figure out which route to take.

Minor suggestion is to change this to "Select one of the following scenarios (states) accordingly:"